assets/css/style.css

```diff
@@ -0,0 +1,96 @@
+:root {
+  --white-color: #fff;
+  --blue-shade-text: #7f8cff;
+  --black-shade-subtitle: #344563;
+  --white-shade-background: #f7f7f9;
+  --default: poppins;
+}
+
+* {
+  font-family: var(--default);
+  padding: 0;
+  margin: 0;
+}
+
+#header {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#header div {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: var(--white-color);
+  width: 100%;
+}
+
+a#my_logo {
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  padding: 30px 20px;
+}
+
+a#img_link {
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 18px;
+  padding: 30px 20px;
+}
+
+#banner_area {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("../images/header-shapes-mobile@2x.svg");
+  background-color: var(--white-color);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  text-align: left;
+  height: 100vh;
+  flex: 1;
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure to reduce the space between the intro paragraph and the navigation bar in order to meet the provided Figma design. :wink: 
- [OPTIONAL] Kindly make sure the title color is the same with the one on the Figma design :wink: 

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/212095615-a48189b0-865b-4fc4-8733-e61389177ccc.png)


   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/212096184-6b88a3af-8af0-4f6a-bd27-1b66d20458d2.png)